utils/symbol.c

```diff
@@ -476,6 +476,8 @@ void sort_dynsymtab(struct uftrace_symtab *dsymtab)
 {
 	unsigned i, k;
 
+	if (dsymtab->nr_sym == 0)
```

Comment:
Due to the ASAN error, malloc(0) actually allocates 1 byte memory and is detected by LeakSan.
I'm not sure this code is really needed.